style.css

```diff
@@ -0,0 +1,129 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@400;500;600;700&family=Poppins:wght@300;400;500&family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,700&display=swap");
+
+:root {
+  --theme-color: #1c1a19;
+  --toolbar-color: #3c3a39;
+  --primary-1: #ff6b00;
+  --primary-4: #fff5e1;
+  --neutral: #dbd8d7;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+}
+
+header {
+  z-index: 1;
+  background: var(--toolbar-color);
+  width: 100%;
+  height: 92px;
+  top: 0;
+}
```

Comment:
- [ ] `z-index`, `height` and `top` properties gravely dents the design of the header. Take a look at the screenshots attached to this code review comment section and notice the difference. Try removing these properties.